test/integration/targets/elb_target/playbooks/roles/elb_lambda_target/tasks/main.yml

```diff
@@ -0,0 +1,101 @@
+---
+- name: set up lambda as elb_target
+
```

Comment:
You need to setup the connection parameters and then use them in each of your tasks, like in:
https://github.com/ansible/ansible/blob/76392649308a51622808b79172c689a3ee41ed24/test/integration/targets/elb_target/playbooks/roles/elb_target/tasks/main.yml#L10-L26

Reply:
I fix the issue with the zip of the lambda source, but I run - **sometimes** into the same assume error you've got.  
 `The role defined for the function cannot be assumed by Lambda`. Run it a few times in a row without deleting the role (and ansible doesn't detect a change), it is accepted. Dunno what here the problem is?




I'm now able to run the integration test: https://gist.github.com/markuman/6a4e7bf57ce78900a4e025c73f8e556c  
But the `elb_target` test fails for me due some region missconfiguration.  
the elb_lambda_target succeeded too, but might fail with the iam_role thing described above and deleting